src/eager/setup.py

```diff
@@ -16,6 +16,9 @@
 parser.add_argument('--additional_libs', default=None, type=str, help='Additional libraries to link against')
 parser.add_argument('--compiler_args', default=None, type=str, help='Additional compiler args to use')
 parser.add_argument('--skip_tests', action='store_true', help='Skips running unit tests as part of the build tests')
+parser.add_argument('--use_preinstalled_torch', action='store_true', help='Use pre-installed torch from the python environment')
```

Comment:
Add check or doc for minimal pytorch version? Master today is at least 1.9.0. ORTModule would have issues with older version, but not sure about eager mode's requirements. Maybe @souptc can comment

Reply:
Fixed